README.md

```diff
@@ -57,112 +51,29 @@ It packs in lots of features that will allow your E-Commerce business to scale i
 **For Developers**:
 Take advantage of two of the hottest frameworks used in this project -- Laravel and Vue.js -- both of which have been used in Bagisto.
 
-### Documentation
-
-#### Bagisto Documentation [https://devdocs.bagisto.com](https://devdocs.bagisto.com)
-
-### Requirements
-
-* **OS**: Ubuntu 16.04 LTS or higher / Windows 7 or Higher (WampServer / XAMPP).
```

Comment:
* [Requirements for Bagisto](https://devdocs.bagisto.com/1.x/introduction/requirements.html)